setup.py

```diff
@@ -60,6 +60,7 @@ def setup_package():
         version=__version__,
         description="A library of spatial analysis functions.",
         long_description=long_description,
+        long_description_content_type="text/x-rst",
```

Comment:
Fixed a warning from the tarball check.

PS: I noticed that there are two README, a markdown and a ResT. PyPI now accepts Markdown, so if you want to join them into one I can update this PR or send a new one later.

Reply:
That would be great!